src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/steps/StartToolStep.groovy

```diff
@@ -296,7 +301,9 @@ class StartToolStep extends Step {
             try {
                  RestApiClientFactory.getRestApiClient(envVars.get('ET_API_HOSTNAME'), envVars.get('ET_API_PORT'), timeout)
             } catch (ApiException e) {
-                throw new TimeoutException("Timeout of ${this.timeout} seconds exceeded for connecting to ${toolName}!")
+                throw new AbortException(
+                        "Timeout of ${this.timeout} seconds exceeded for connecting to ${toolName}!" +
+                        "Please ensure the tool is running and its API endpoint is accessible.")
```

Comment:
If the tool is started correctly, the endpoint can be reached. The user has no possibility to check the endpoint separately